app/src/pages/inside/productVersionPage/linkedTestCasesTab/tagList/tagList.jsx

```diff
@@ -46,21 +47,56 @@ export const TagList = ({ tags }) => {
     setCount(overflowedElements);
   };
 
+  const getFullWidthOffset = () => {
+    const parentElement = listRef.current;
+    const hiddenSet = new Set();
+
+    if (!parentElement) return;
+
+    const tagElements = [...parentElement.children].filter(
+      (child) => !child.classList.contains('tag-list__item--button'),
+    );
+
+    if (tagElements.length === 0) return;
```

Comment:
```suggestion
    if (isEmpty(tagElements)) return;
```

Reply:
I dont see a reason to create additional variable for this condition that used only once

Reply:
It is not variable, it is util function from lodash. We have it installed already `lodash.is-empty`